app/src/main/java/com/codesoom/assignment/controllers/TaskRepository.java

```diff
@@ -0,0 +1,77 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * 할 일을 관리하는 클래스이다.
```

Comment:
클래스에 대한 설명을 적는데 클래스라고 적으면 어색하겠죠 ㅎㅎ 한편 아직 `Repository` 단어에 대한 학습도 필요할 것 같습니다. (조사해 보세요)
이렇게 해보면 어떨까요?
```suggestion
 * 할 일 정보를 관리합니다.
```

Reply:
스프링의 계층을 조사해보았습니다.

1. 프레젠테이션 계층 (Controller)
- 웹 클라이언트의 요청 및 응답을 처리한다.

2. 서비스 계층 (Service)
- 비즈니스 로직과 도메인 모델의 적합성을 검증한다.

3. 데이터 엑세스 계층 (Repository)
- 데이터에 직접 접근하거나 특정 데이터베이스, 파일을 획득하여 CRUD를 할 수 있다.